code-of-conduct.md

```diff
@@ -2,48 +2,50 @@
 
 denhac does not promote or discriminate against any person, population group, or organization with regard to categories protected by applicable United States law. These include, but are not limited to race, color, religion, sex, gender expression, sexuality, physical appearance, language, education background, national origin, age, disability, and veteran status.
 
-denhac strives to create an inclusive environment where everyone can feel welcome and respected in our space. We know that, while there is not a formal hierarchy here as in a workplace, everyone comes into our space with their own different histories and social norms. We expect you to both communicate with each other about the boundaries/limits you wish to hold, and respect when someone else communicates a boundary with you. If a boundary is clearly expressed to you (whether in person, over Slack messages, or via other means), and you deliberately overstep that boundary, it will be treated as a violation of our anti-harassment policy. We believe that initial communication of a boundary can be done without board support in most cases, but please reach out to any member of the Board of Directors if you would like support of any kind in any phase of the process of boundary setting. 
+denhac strives to create an inclusive environment where everyone can feel welcome and respected in our space. While there is not a formal hierarchy here as in a workplace, everyone comes into our space with their own different histories and social norms. We on the Board of Directors expect members to both communicate with each other about the boundaries/limits you each wish to hold, and respect when someone else communicates a boundary with you. If a member deliberately oversteps a boundary that is clearly expressed to them (whether in person, over Slack messages, or via other means) it will be treated as a violation of our anti-harassment policy and the board may need to take disciplinary action, if relevant. Initial communication of a boundary can be done without direct board member support in most cases, but please reach out to any member of the Board of Directors if you would like support of any kind in any phase of the process of boundary setting.
```

Comment:
> oversteps a boundary that is clearly expressed to them

I feel like this should be `that has been clearly expressed to them`

> the board may need to take disciplinary action

I think it would be more appropriate to strike `need to` and simply state `the board may take disciplinary action`